snippets/header-drawer.liquid

```diff
@@ -139,9 +139,9 @@
                 -%}
               </a>
             {%- endif -%}
-            {%- if section.settings.enable_country_selector or section.settings.enable_language_selector -%}
+            {%- if localization.available_countries or localization.available_languages -%}
               <div class="menu-drawer__localization header__localization">
-                {%- if section.settings.enable_country_selector and localization.available_countries.size > 1 -%}
+                {%- if localization.available_countries and localization.available_countries.size > 1 -%}
```

Comment:
It is a general behavior. This was implemented so that if a merchant decides to uncheck the currency/language picker  in the header and keep it in the announcement bar, the localization form will appear in the mobile-menu drawer. I included a video above to show this behavior.

Reply:
hmmm this is a valid point. We can always add a separate issue to address removing the locales and/or social media icons in the menu drawer if a merchant wants neither to show up anywhere on there store :)

Reply:
That's a good call out Ludo. I think it's still fair for us to keep it in the mobile drawer by default, even if a merchant COULD have an app and want to turn it off in the future. Let's prioritize solving for the majority.

1. Most new merchants start with Dawn (and shouldn't have to use an app)
2. Many apps like lang/currency picker install apps using JS/code insertions, which I believe takes merchants off the upgrade path anyways
3. IF a merchant COULD upgrade their Dawn version AND preferred using another app, we can provide a quick code snippet to hide the native mobile UI

Reply:
⚠️ We should try and make sure to flag it in the release notes ⚠️